library/Ivoz/Provider/Domain/Model/Administrator/Administrator.php

```diff
@@ -191,6 +191,10 @@ public function unserialize($serialized)
 
     protected function sanitizeValues(): void
     {
+        if (empty($this->getPass())) {
+            $this->setActive(false);
+        }
+
```

Comment:
It's nice to check this in the frontend, but we need to enforce all the requested rules also in the back.

I don‘t see any validation for User in the backend.